src/features/script-load.js

```diff
@@ -46,3 +46,17 @@ systemJSPrototype.instantiate = function (url, firstParentUrl) {
   });
 };
 
+if (typeof window !== 'undefined') {
+  window.addEventListener('load', loadScriptModules);
+}
+
+function loadScriptModules() {
+  window.removeEventListener('load', loadScriptModules);
+  document.querySelectorAll('script[type=systemjs-module]').forEach(function (script) {
+    if (script.src) {
+      System.import(script.src.slice(0, 7) === 'import:' ? script.src.slice(7) : script.src);
```

Comment:
Strictly speaking, `script.src` should be passed to `new URL` to avoid it resolving as a bare specifier in import maps.

Reply:
Sorry, not `URL`, we have an internal `resolveUrl` function which can be used for IE11 support.

Reply:
👍 updated